### PR TITLE
Debugger: Fix typos in code comments

### PR DESCRIPTION
**Pull request by nomadbyte**
This is an excellent and quite complex plugin. Fixing typos in the code comments helps new devs to get familiar with the plugin's design and logic. Well, it helped me at least...

### CODE REVIEW

File: debugger/src/watch_model.c [hunk 1]
```diff
@@ -265,7 +265,7 @@ void change_watch(GtkTreeView *tree, GtkTreeIter *iter, gpointer var)
 	/* update variable */
 	update_variable(store, iter, v, FALSE);
 
-	/* if item have children - remove them */ 		
+	/* if item have children - remove them */
```
Comment (elextr) [inline debugger/src/watch_model.c:268]: has

**Comment by elextr** [comment]
LGTM

File: debugger/src/watch_model.c (hunk 1 above)
Comment (nomadbyte) [reply debugger/src/watch_model.c]: yep, another one!
pushed the updates.

**Comment by elextr** [comment]
Helpful comments, who would have thought ... @nomadbyte thanks.

**Pull request merged**